pyproject.toml

```diff
@@ -9,6 +9,8 @@ python = "^3.7"
 Flask = "^2.0.3"
 python-dotenv = "^0.14.0"
 requests = "^2.28.1"
+DateTime = "^4.5"
```

Comment:
I don't think this package is used. I can see you importing `datetime` in `item.py` but that is a built-in python module.

Please can you run the following command and then check that your app still works?

`poetry remove DateTime`

Reply:
It still works, I'll make the push the change to Github in the next PR.